views/home.ejs

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html> 
+<html> 
+<head> 
+    <title>Sample EJS</title>
+</head> 
+<body> 
+    <p>This is a EJS template.</p>
+    <p>read more @ https://ejs.co/</p>
+</body> 
+</html>  
```

Comment:
load this somewhere in some file to ensure that this works.

Reply:
added partial file for demo